drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -623,6 +623,11 @@ static void set_up_fixed_clock_sources(void)
 		z_stm32_hsem_lock(CFG_HW_RCC_SEMID, HSEM_LOCK_DEFAULT_RETRY);
 
 #if defined(PWR_CR_DBP) || defined(PWR_CR1_DBP) || defined(PWR_DBPR_DBP)
+#if defined(CONFIG_SOC_SERIES_STM32F1X)
+		/* Set the PWREN and BKPEN bits in the RCC_APB1ENR register */
+		LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_PWR);
+		LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_BKP);
+#endif
```

Comment:
Please move to `drivers/clock_control/clock_stm32f1.c` in function `config_enable_default_clocks()` under `STM32_LSE_ENABLED` condition.

Reply:
Do we need to consider the impact of `z_stm32_hsem_lock here`?
I don't think there's any guarantee that other processes won't turn off the clock under non-atomic operations.

Reply:
> Do we need to consider the impact of z_stm32_hsem_lock here?

Not for F1 series